docs/framework/wcf/how-to-implement-a-wcf-contract.md

```diff
@@ -105,13 +105,14 @@ Namespace GettingStartedLib
 End Namespace
 ```
 
-Each method implements the calculator operation and writes some text to the console to make testing easier.
+Each method implements the calculator operation and writes text to the console to make testing easier.
 
 ## Example
```

Comment:
I feel like this whole section is not really needed. It's just repeating the previous code sample. Instead, I would make a note that this service contract is implementing the interface designed in the (previous tutorial step|LINK).